setup.py

```diff
@@ -31,8 +31,9 @@ def package_data(pkg, roots):
         'launchcontainer',
     ],
     install_requires=[
```

Comment:
@briandant As far as I know setting such explicit requirements for this XBlock is going to break it on either Ficus or Ginkgo.

I'm not sure if this is completely OK, but I'd suggest that we either:

 A. Specifiy the requirements without versions, or
 B. Remove the the requirements altogether. 

What do you think?